server/urls.py

```diff
@@ -16,10 +16,34 @@
 from django.contrib import admin
 from django.urls import path, include 
 
+
+
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+
+# for testing
+schema_view = get_schema_view(
+   openapi.Info(
+      title="teesco-API",
+      default_version='v1',
+      description="Test description",
+      terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="contact@snippets.local"),
+      license=openapi.License(name="BSD License"),
+   ),
+   public=True,
+   permission_classes=(permissions.AllowAny,),
+)
+
+
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/query/',include('query.urls')),
     path('api/users/',include('users.urls')),
     path('api/org/',include('org.urls')),
     path('api/tasks/',include('tasks.urls')),
+    path('swagger', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui')
```

Comment:
```suggestion
    path('', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui')
```

Lets keep this swagger docs in the home route

Reply:
okay sir